content/browser/secure_embed_connector_impl.cc

```diff
@@ -180,6 +200,11 @@ SecureEmbedConnectorImpl::GetRootRenderWidgetHostView() {
   return GetParentRenderWidgetHostView();
 }
 
+void SecureEmbedConnectorImpl::OnRenderViewReady() {
+  // When the RenderView is ready, update the view in case it has changed.
```

Comment:
Here or in the header, it is worth documenting that this is needed for the initial navigation.